configure.ac

```diff
@@ -19,7 +19,8 @@ AC_INIT(
     [https://github.com/jgeboski/bitlbee-steam/issues],
     [bitlbee-steam],
     [https://github.com/jgeboski/bitlbee-steam],
-    []
+    [gtk-doc],
+    [gtk_doc_version]
```

Comment:
Also a relict from me messing around with different configs. The gtk-doc documentation stated to include this: https://github.com/GNOME/gtk-doc/blob/master/configure.ac
And:
https://developer.gnome.org/gtk-doc-manual/stable/settingup_autoconf.html.en